### PR TITLE
Gpu support

### PR DESCRIPTION
**Pull request by alpha-carinae29**
This PR will add a docker file and instruction to run Smart Social Distancing on x86 nodes with GPU.

### CODE REVIEW

**Review by mats-claassen — COMMENTED** [review]
Nice PR. I tested this on a GTX 1070 and it seems to work. I get logs like
`Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 7036 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1070 ...`

However I am not getting any speedups compared to running it without GPU. Have you seen speedups with these models and another GPU?

File: README.md [hunk 1]
```diff
@@ -214,6 +214,24 @@ docker build -f x86.Dockerfile -t "neuralet/smart-social-distancing:latest-x86_6
...
+# If you use the OpenPifPaf model, download the model first:
+./download-x86-openpifpaf-model.sh
+
+# If you use the MobileNet model run this instead:
+# ./download_x86_model.sh
+
+# 1) Build Docker image (This step is optional, you can skip it if you want to pull the container from neuralet dockerhub)
+docker build -f x86-gpu.Dockerfile -t "neuralet/smart-social-distancing:latest-x86_64_gpu" .
+
+# 2) Run Docker container:
```
[the remark this answers is not in the document]
Comment (alpha-carinae29) [reply README.md:230]: done

**Comment by alpha-carinae29** [comment]
> Nice PR. I tested this on a GTX 1070 and it seems to work. I get logs like
> `Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 7036 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1070 ...`
> 
> However I am not getting any speedups compared to running it without GPU. Have you seen speedups with these models and another GPU?

Thank you @mats-claassen for mentioning this, I had tested the GPU support for SSD detector only and not for OpenPifPaf. Now it is fixed for OpenPifPaf too, can you test it one more time please?

**Comment by mats-claassen** [comment]
Works much better now. I can process multiple frames per second with OpenPifPaf.

What about Mobilenet? if that is not supported for GPU we could remove it from the comment in the .ini file

**Comment by alpha-carinae29** [comment]
Both Mobilenet and OpenPifPaf are currently supported for GPU.

**Review by renzodgc — APPROVED** [review]
Looks good to me!

**Review by mats-claassen — APPROVED** [review]

**Pull request merged**